server/helpers/Token.js

```diff
@@ -0,0 +1,41 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+
+config();
+/**
+ * @class Token
+ */
+class Token {
+  /**
+   * @static
+   * @param {user} user object from the database
+   * @returns {string} returns thetoken
+   * @memberof Token
+   */
+  static async create(user) {
+    const { id, email } = user;
+    const token = await jwt.sign({
+      userId: id,
+      email,
```

Comment:
how long does this token take before expiring?

You could place this value as an environment variable (and give a default value if one is not supplied)

Reply:
We concluded not to let the token expire for now so that we won't be doing a lot of changing token in development.

Reply:
then you do not understand my comment then.
put the value of expiration as an environment variable (this way, it's not encoded and everyone can modify as needed

Reply:
ok. I understand better. Thanks!